packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -57,7 +57,7 @@ The app logo that you submit to the OIN Manager in the **General Settings** > **
 
 * If your logo isn't square, consider using your website favicon. Alternatively, you can use the first letter of your app wordmark and convert it to a square image.
 
-* You can use a slightly non-square icon, but consider the scaling that occurs when your logo is resized to fit a 200 x 200 pixel area. If you have more logo questions, contact the [OIN team](mailto:oin@okta.com).
+* You can use a slightly non-square icon, but consider the scaling that occurs when your logo resizes to fit a 200 x 200 pixel area. If you have more logo questions, contact the [OIN team](mailto:oin@okta.com).
```

Comment:
Do we really need to offer the oin@ handle here? "If you have more logo questions, contact the [OIN team](mailto:oin@okta.com)."

Recommend either putting more detail around why ISVs would email further beyond this logo instruction, or just remove that sentence.